fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -133,6 +159,17 @@ def self.available_options
                                        verify_block: proc do |value|
                                          available = ['ios', 'appletvos', 'mac']
                                          UI.user_error!("Invalid platform '#{value}', must be #{available.join(', ')}") unless available.include?(value)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :dsym_worker_threads,
+                                       env_name: "FL_UPLOAD_SYMBOLS_TO_CRASHLYTICS_DSYM_WORKER_THREADS",
+                                       default_value: '1',
+                                       optional: true,
+                                       description: "The number of threads to use for simultaneous dSYM upload",
+                                       verify_block: proc do |value|
+                                         min_threads = 1
+                                         max_threads = 15
+                                         UI.user_error!("Too few threads (#{value}) minimum number of threads: #{min_threads}") unless Integer(value) >= min_threads
```

Comment:
If you define the type, you don't need the `Integer` part

Reply:
Fixed.